pkcs7_test.go

```diff
@@ -375,3 +372,32 @@ func UnmarshalTestFixture(testPEMBlock string) TestFixture {
 
 	return result
 }
+
+type certChain struct {
```

Comment:
This file contains testing helper functions used throughout other testing files, so I opted to add some new helper functions for the new verify work here. Happy to move them to `verify_test.go` as well.